cosmos-admin/get_user_stats.sh

```diff
@@ -0,0 +1,114 @@
+#! /bin/bash
+# Copyright 2015 Telefonica Investigacion y Desarrollo, S.A.U
+#
+# This file is part of fiware-cosmos.
+#
+# fiware-cosmos is free software: you can redistribute it and/or
+# modify it under the terms of the GNU Affero General Public License as
+# published by the Free Software Foundation, either version 3 of the
+# License, or (at your option) any later version.
+#
+# fiware-cosmos is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU Affero
+# General Public License for more details.
+#
+# You should have received a copy of the GNU Affero General Public License
+# along with fiware-cosmos. If not, see http://www.gnu.org/licenses/.
+#
+# For those usages not covered by this license please contact with
+# francisco dot romerobueno at telefonica dot com
+
+# Show the usage
+if [ $# -ne 6 ]; then
+        echo "Usage: ./get_user_stats.sh <db_host> <db_port> <db_name> <db_user> <db_password> <ssh_logs>"
+        exit 1;
+fi
+
+# Arguments
+dbHost=$1
+dbPort=$2
+dbName=$3
+dbUser=$4
+dbPassword=$5
+sshLogs=$6
+
+# From string to number months converter
+# $1 --> month to be converted
+convert_month() {
+	local month_str=$1
+
+	case $month_str in 
+                Jan )
+                        return 1
```

Comment:
Fixed in https://github.com/telefonicaid/fiware-cosmos/commit/5c07eed2af6cdb9c7cb9b61aaf34991335670db2